README.md

```diff
@@ -15,6 +15,10 @@ thousands of repositories in a single node. Stay tuned!
 
 ## Examples
 
+To see the SQL subset currently supported take a look at [this list](https://github.com/src-d/go-mysql-server/blob/master/SUPPORTED.md) from [src-d/go-mysql-server](https://github.com/src-d/go-mysql-server).
```

Comment:
Should we add a link to the same commit as is used by the project? to avoid misunderstandings between the actual go-mysql-version and the go-mysql-version that is in use on gitbase.

Reply:
Sure